packit_service/worker/jobs.py

```diff
@@ -98,7 +98,7 @@ def get_handlers_for_event(
 
 def get_config_for_handler_kls(
     handler_kls: Type[JobHandler], event: Event, package_config: PackageConfig
-) -> Optional[JobConfig]:
+) -> List[JobConfig]:
     """
     Get a JobConfig relevant to event and the handler class.
```

Comment:
This is not "a JobConfig" but multiple ones. Please update the doc-string to reflect the change in the return type and explain whether the order of the list has any meaning or not.

Reply:
> This is not "a JobConfig" but multiple ones.

Thanks, I'll update that.

> Please update the doc-string to reflect the change in the return type and explain whether the order of the list has any meaning or not.

It's a list => ordering has meaning. (I am using `set` where the order is not important.) I'll make a note to the doc as well.